akka-docs/src/main/paradox/stream/operators/Source-or-Flow/zipLatest.md

```diff
@@ -0,0 +1,33 @@
+# zipLatest
+
+Combines elements from each of multiple sources into @scala[tuples] @java[*Pair*] and passes the @scala[tuples] @java[pairs] downstream, picking always the latest element of each.
+
+@ref[Fan-in operators](../index.md#fan-in-operators)
+
+@@@div { .group-scala }
+
+## Signature
+
+@@signature [Flow.scala]($akka$/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #zipLatest }
+
+@@@
+
+## Description
+
+Combines elements from each of multiple sources into @scala[tuples] @java[*Pair*] and passes the @scala[tuples] @java[pairs] downstream, picking always the latest element of each.
```

Comment:
yes I would tend to agree, however I followed the same convention as for 'zip.md' and 'zipWith.md' where the same paragraph is repeated between summary and description. So I guess either we would have to eliminate the description section (but it would probably look strange in the documentation) or come up with a more detailed explanation in the description?